src/app/teams/teams.service.ts

```diff
@@ -98,6 +98,13 @@ export class TeamsService {
     );
   }
 
+  deleteTeam(team) {
+    return this.updateTeam({ ...team, status: 'archived' })
+      .pipe(
```

Comment:
This pipe/switchMap is not necessary.  `newTeam` already has all of the properties on `team`